### PR TITLE
Fixed Mobile NavBar text and Translation .

### PR DESCRIPTION
**Pull request by Pratham-Gaikwad**
**Fixed Issues** 
1. The menu items are not capitalized
2. When switching to portugiese, the text doesnt change.

### CODE REVIEW

File: src/app/components/NavBar.tsx [hunk 1]
```diff
@@ -332,6 +332,7 @@ function NavbarItemMobile({
   setMenuOpen,
 }: NavbarItemMobileProps) {
   const active = target === usePathname();
+  const translate = useTranslations();
```
Comment (dcts) [inline src/app/components/NavBar.tsx:335]: Please change to follow our conventions:
```suggestion
  const t = useTranslations();
```

File: src/app/components/NavBar.tsx [hunk 2]
```diff
@@ -343,7 +344,7 @@ function NavbarItemMobile({
           active ? "text-[#cafc40]" : "text-white"
         }`}
       >
-        {title}
+        {translate(title)}
```
Comment (dcts) [inline src/app/components/NavBar.tsx:347]: Please change to follow our conventions:
```suggestion
        {t(title)}
```

**Review by dcts — CHANGES_REQUESTED** [review]

**Comment by Pratham-Gaikwad** [comment]
requested changes have been done

**Review by Pratham-Gaikwad — COMMENTED** [review]
requested changes have been done

**Review by dcts — APPROVED** [review]

**Comment by dcts** [comment]
Thanks for the fix @Pratham-Gaikwad 🙏

**Pull request merged**